packages/create-sitecore-jss/src/templates/nextjs-sxa/src/pages/api/sitemap.ts

```diff
@@ -34,12 +34,10 @@ const sitemapApi = async (
     res.setHeader('Content-Type', 'text/xml;charset=utf-8');
 
     // need to prepare stream from sitemap url
-    return new AxiosDataFetcher()
-      .get(sitemapUrl, {
-        responseType: 'stream',
```

Comment:
I think we lost "stream" processing, since the response type is "stream"
did you verify that?

Reply:
i have added the stream processing logic.